docker-ci/Dockerfile

```diff
@@ -56,6 +56,7 @@ RUN apt update \
     libssl-dev \
     build-essential \
     protobuf-compiler \
+    libsqlite3-dev \
```

Comment:
I think with the `bundled` feature in `rusqlite` maybe you can remove this safely from the Docker build... I'm not sure 100% but is wothy to try...

Reply:
ok yes you were right I have reverted the CI version and passing CI